Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/CursorContextManipulationHandler.cs

```diff
@@ -36,8 +36,9 @@ private void Start()
         private void ManipulatorHoverEntered(ManipulationEventData manipEvent)
         {
             hoverCount++;
+            var mh = manipEvent.ManipulationSource.GetComponent<ManipulationHandler>();
```

Comment:
> var mh = manipEvent.ManipulationSource.GetComponent<ManipulationHandler>(); [](start = 11, length = 76)

Not really related to your current change, but do you know if this mh is always gonna be the same as the mh defined in Start()?

i.e. should we just cache mh in Start and use that to then do mh.ManipulationType?

I assume that what this code here is trying to do is ask questions about the manipulation handler on the same game object, not about the manipulation handler on some unrelated game object.